cgmes/cgmes-measurements/src/main/java/com/powsybl/cgmes/measurements/CgmesMeasurementsModel.java

```diff
@@ -29,26 +31,17 @@ public CgmesMeasurementsModel(TripleStore tripleStore) {
     }
 
     public PropertyBags analogs() {
-        LOG.info("Querying triplestore for Analogs");
-        return queryTripleStore("analogs");
+        LOG.trace("Querying triplestore for Analogs");
+        return queryTripleStore("analogs", queryCatalog, tripleStore);
     }
 
     public PropertyBags bays() {
-        LOG.info("Querying triplestore for Bays");
-        return queryTripleStore("bays");
+        LOG.trace("Querying triplestore for Bays");
+        return queryTripleStore("bays", queryCatalog, tripleStore);
     }
 
     public PropertyBags discretes() {
-        LOG.info("Querying triplestore for Discretes");
-        return queryTripleStore("discretes");
-    }
-
-    private PropertyBags queryTripleStore(String queryKey) {
-        String query = queryCatalog.get(queryKey);
-        if (query == null) {
-            LOG.warn("Query [{}] not found in catalog", queryKey);
-            return new PropertyBags();
-        }
-        return tripleStore.query(query);
+        LOG.trace("Querying triplestore for Discretes");
```

Comment:
we could put that log in the `TripleStoreUtils::queryTripleStore`, and then we'll have the log also when querying from `CgmesShortCircuitModel`.